### PR TITLE
rromeo

### PR DESCRIPTION
**Pull request by Bisaloo**
I'm not super satisfied with the flow in the two `Developing an API package` but I need to take a step back. I'll have another look tomorrow morning. 

Feel free to have a look in the mean time if you wish though. We don't expect major changes from our side at this point.

@Rekyt, you also need to [create your author page](https://github.com/ropensci/roweb2#create-an-authors-index-file-for-yourself)

Specific points to discuss: 

- should we link to the CRAN page or the GitHub repo when we talk about packages?

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Your thoughts in this post are so well-expressed. Of course the introduction captured my heart ;-)

I have a few comments - all minor - at this stage
- when referring to a person, refer to them by name and link to 1) their rOpenSci author page or 2) their  Twitter or GitHub, without showing their handles in the text
- remove hyphen from `open-access` in one (or more?) occurrence
- scrapping -> scraping
- environmental variable -> environment variable
- consider linking to any rOpenSci packages you mention in their first occurrence
- where you welcome contributions, consider opening an issue or two with label `help wanted` and you could link to them. (I'm happy to tweet-promote any help-wanted issues that package authors would like to have highlighted. No obligation)
- in paragraph  "A similar issue lied..." there are a couple of issues with English usage
- please link to the open review thread somewhere if you don't mind

Really excited to publish this.

**Comment by Bisaloo** [comment]
Thanks for your comments! f2f26c3 should address most of them.

About

> consider linking to any rOpenSci packages you mention in their first occurrence

could you link to a specific place where we forgot to do it please? It should already be done in most cases.

File: content/blog/2019-06-04-rromeo.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+### SHERPA/RoMEO API
+
+SHERPA/RoMEO has been available through a web interface since at least February 2004 according to the [Internet Archive](https://archive.org/) and their API was released in December 2006 (from the Internet Archive <https://web.archive.org/web/20061211125806/http://www.sherpa.ac.uk:80/romeo.php>), making their data available to anyone since then.
+
+There are three versions of the manuscript considered in SHERPA/RoMEO:
+
+1. pre-print, which is the manuscript version **before peer review**;
+1. post-print, which is the manuscript as **accepted after peer review** but not yet typeset as an article in the journal;
+1. and typeset manuscript/publisher's manuscript, which is the manuscript with the text after peer-review, **fully typeset**, as it appears in the journal.
+
```
Comment (Bisaloo) [inline content/blog/2019-06-04-rromeo.md]: Hi @stefaniebutland :wave:, we would like to add an image here to explain more clearly the difference between pre-prints, post-prints and publisher's pdf. We have identified two good visuals but we can't choose between them. An external opinion would be great!

https://paleorxiv.github.io/article_types.jpg (CC BY)

or

https://s3.amazonaws.com/libapps/accounts/3099/images/Publishingprocess2.PNG (public domain)

What do you think?

Comment (stefaniebutland) [reply content/blog/2019-06-04-rromeo.md]: I like the clarity of the first one

**Comment by stefaniebutland** [comment]
This was my error. They are fine as is

>> consider linking to any rOpenSci packages you mention in their first occurrence

> could you link to a specific place where we forgot to do it please? It should already be done in most cases.

**Comment by stefaniebutland** [comment]
@Bisaloo @Rekyt I will be offline from now until Mon Jun 3. I'll do a final check then and publish and tweet on the 4th. Cheers!

**Comment by Rekyt** [comment]
Thank you @stefaniebutland for your comments! Alright we shouldn't change the article much a part from including the figure.

Great for the "help wanted" tweet!
Cheers

**Pull request merged**